.ci/jobs/packaging.yml

```diff
@@ -4,6 +4,7 @@
     display-name: Beats Packaging
     description: Make the packages for beats and publish them on a GCS bucket.
     view: Beats
+    disabled: true
```

Comment:
Nit: Should we add a comment here explaining why this is disabled?

Reply:
I will revert it in a few, as soon as I fix the issue